src/renderers/__tests__/EventPluginHub-test.js

```diff
@@ -24,6 +24,8 @@ describe('EventPluginHub', () => {
   });
 
   it('should prevent non-function listeners, at dispatch', () => {
+    // ReactDOMFiberComponent warns for non-function event listeners
+    spyOn(console, 'error');
```

Comment:
Done in https://github.com/facebook/react/pull/10453/commits/c377e352ce33f233cc66aa76ead517f504c79f73